sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationClient_private.cs

```diff
@@ -63,9 +64,11 @@ internal static void BuildBatchQuery(RequestUriBuilder builder, SettingSelector
                 builder.AppendQuery(LabelQueryFilter, selector.LabelFilter);
             }
```

Comment:
Can this `BuildBatchQuery` method and associated tests be deleted? It seems it's only referenced in tests, so we don't use it internally but it's marked as `internal`. Not sure if I'm missing something.

Reply:
I don't know of the history there or what it was used for.   If we don't reference it outside of tests, and those tests aren't tied to public API scenarios, it seems safe to delete it.

@ShivangiReja: Thoughts?

Reply:
Looks like these are just the unit test cases and we are not using `BuildBatchQuery` method in our code base. Therefore, we can safely remove this method.

Reply:
It seems we forgot to remove this method [here](https://github.com/Azure/azure-sdk-for-net/pull/30223/files#diff-cdde486ece8098d6e080b2d4a2eb6f3b4bca8bc9377a7590f25710bdf0f57e4a). Currently, this method is only used for the unit test cases and is not used in our codebase. Therefore, we can safely remove it.